control/iosys.py

```diff
@@ -706,7 +768,6 @@ def _out(self, t, x, u):
             + np.dot(self.D, np.reshape(u, (-1, 1)))
         return np.array(y).reshape((-1,))
 
-
```

Comment:
I think this line removal will generate a PEP8 error (check with pycodestyle).  Two blank lines are required before function and class definitions.